rotkehlchen/data/importer.py

```diff
@@ -784,6 +784,74 @@ def import_blockfi_trades_csv(self, filepath: Path) -> Tuple[bool, str]:
                     return False, str(e)
         return True, ''
 
+    def _consume_binance_purchases(self, csv_row: Dict[str, Any]) -> None:
+        """
+        Consume the file containing only purchases by credit/debit card from Binance.
```

Comment:
Can you add, just like the other functions here in the same file what exceptions can be raised by this function? Example: https://github.com/rotki/rotki/blob/cae712a9f8934a1168e23f6f26f4b3c465dcfb39/rotkehlchen/data/importer.py#L92-L100